### PR TITLE
Fortune Teller, Priestess, Warlock & Tanner are now a thing

### PR DESCRIPTION
**Pull request by Randium**

### CODE REVIEW

**Comment by Randium** [comment]
These builds failed because... an alternate version came in quicker? 🤔

File: main.py [hunk 1]
```diff
@@ -69,12 +69,12 @@
...
     isGameMaster = False
     if message.guild == gamelog_channel.guild:
-        if game_master in [y.id for y in message.channel.guild.get_member(message.author.id).roles]:
+        if game_master in [y.id for y in message.author.roles]:
             isGameMaster = True
 
     isAdmin = False
     if message.guild == gamelog_channel.guild:
-        if administrator in [y.id for y in message.channel.guild.get_member(message.author.id).roles]:
+        if administrator in [y.id for y in message.author.roles]:
```
Comment (Bentechy66) [inline main.py]: this will not work in a `PrivateChannel`

Comment (Randium) [reply main.py]: That's why the line above make sure that line will never be triggered in a DM.

Comment (Randium) [reply main.py]: *makes

Comment (Bentechy66) [reply main.py]: I'm unsure if a `PrivateChannel` is a channel with permissions set, the API docs don't make it clear.

**Pull request merged**